mora/converters/reading.py

```diff
@@ -267,8 +267,8 @@ def convert_address(obj):
                 'location': _get_location(info.location),
                 'visibility': {
                     'user-key': info.visibility,
-                    'uuid': meta.PHONE_VISIBILITY_UUIDS[info.visibility],
-                    'name': meta.PHONE_VISIBILITIES[info.visibility],
+                    'uuid': meta.PHONE_VISIBILITY_UUIDS.get(info.visibility),
+                    'name': meta.PHONE_VISIBILITIES.get(info.visibility),
```

Comment:
Are we still using things from this file for the new API? If so, we should extract the relevant methods so we can easily this kill this entire file along with the rest of the legacy API when the time is right.

Reply:
No, but the changes to the fixtures causes us to have ‘visibilities’ which aren't, in fact, that. So it was just extra error handling for now. Probably should've mentioned that in the commit…